stl/inc/xmemory

```diff
@@ -97,7 +97,8 @@ struct _Default_allocate_traits {
 #ifdef __clang__ // Clang and MSVC implement P0784R7 differently; see GH-1532
         _CONSTEXPR20
 #endif // defined(__clang__)
-        void* _Allocate(const size_t _Bytes) {
+        void*
+        _Allocate(const size_t _Bytes) {

```

Comment:
These two occurrences are unfortunate, but not worth blocking the PR over. I'll experiment and see if I can find an improvement.

Reply:
Agreed, thanks.